README.md

```diff
@@ -64,13 +74,30 @@ dfx start [--clean] [--background]
 II_ENV=development dfx deploy --no-wallet --argument '(null)'
 ```
 
-Then, run `CANISTER_ID=$(dfx canister id internet_identity) npm start` to start webpack-dev-server.
+To serve the frontend locally via webpack (recommended during development), run
+the following:
+
+```bash
+CANISTER_ID=$(dfx canister id internet_identity) npm start
```

Comment:
I don't need to do the `CANISTER_ID` bit on my end. If you've run `dfx deploy` beforehand (to install the canister) the canisterId should be available and picked up by the webpack config from `.dfx/local/canister_ids.json` automatically.

Reply:
You're correct. Amending.